workers/python/python/extraction/docling.py

```diff
@@ -51,7 +51,7 @@ async def docling_extraction(file):
             dict = doc.export_to_dict()
             markdown = doc.export_to_markdown()
             chunks = [{"text": c.text}
-                      for c in list(HierarchicalChunker().chunk(doc))]
+                      for c in list(HybridChunker(tokenizer="BAAI/bge-small-en-v1.5").chunk(doc))]
```

Comment:
this model is loaded in memory? We could host it perhaps in future or use our bge-large